packages/core/__tests__/utils.ts

```diff
@@ -69,17 +69,24 @@ export function makeConfigForRuleset(
   });
 }
 
-export async function makeConfig(
-  rules: Record<string, RuleConfig>,
-  decorators?: Record<string, DecoratorConfig>,
-  configPath?: string
-) {
+export async function makeConfig({
```

Comment:
@tatomyr I changed this helper function arguments to make it flexible and not pass params in special order and also to handle Arazzo rules case - as it does not merge `rules` with `ArazzoRules` as other specs.